addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -631,7 +631,7 @@ export class LinkPlugin extends Plugin {
             }
         } else {
             const closestLinkElement = closestElement(selection.anchorNode, "A");
-            if (closestLinkElement && closestLinkElement.isContentEditable) {
+            if (closestLinkElement) {
```

Comment:
@visp-odoo I don't think removing ```isContentEditable``` condition is right solution, in case of Navbar links we have to handle the case, this issue will occur on links where anchor have span some other editable element, so I guess we need   should check whether closestAnchor have editable child then we should allow further.

Previously Navbar links were explicitly handled.